desktop-environment.el

```diff
@@ -41,6 +41,32 @@
   :group 'environment)
 
 
+;;; Customization - keyboard backlight
+
+(defcustom desktop-environment-keyboard-backlight-set-command "SetBrightness"
+  "Shell command setting the keyboard backlight.
```

Comment:
the docstring doesn't match the value: it's not a shell command anymore but a D-Bus method :-). Moreover, I don't see the point of having a D-Bus method as a `defcustom` because there is no way the user could change the value of the variable and get something working.

I suggest you remove this variable entirely. At a later point, we can always implement what I suggested earlier regarding the function vs. shell command `defcustom`.

Reply:
I agree. It would be ideal to use a defcustom of the D-Bus method.
Removed all the defcustom except for 'normal-[increment,decrement]'.